src/frontend/screens/DownloadManager/components/DownloadManagerItem/index.tsx

```diff
@@ -151,8 +152,24 @@ const DownloadManagerItem = observer(({ element, current, state }: Props) => {
   const handleSecondaryActionClick = () => {
     if (state === 'paused') {
       window.api.resumeCurrentDownload()
+      trackDownloadStatusChange({
+        event: 'Game Install Resumed',
+        properties: {
+          store_name: runner,
+          game_title: gameInfo.title,
+          game_name: gameInfo.app_name
+        }
+      })
     } else if (state === 'running') {
       window.api.pauseCurrentDownload()
+      trackDownloadStatusChange({
+        event: 'Game Install Paused',
+        properties: {
+          store_name: runner,
+          game_title: gameInfo.title,
+          game_name: gameInfo.app_name
+        }
+      })
```

Comment:
we're going to remove these now that we're tracking this in the download manager right?